### PR TITLE
javascript-local-storage - Kyle Kettler

### PR DESCRIPTION
**Pull request by kyle-kettler**
Kapture 2023-05-26 at 16 46 03

### CODE REVIEW

File: javascript-local-storage/javascript-local-storage-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- How do you store data in `localStorage`?
+
+- How do you retrieve data from `localStorage`?
+
+- What data type can `localStorage` save in the browser?
+
+- When does the `'beforeunload'` event fire on the `window` object?
```
Comment (shawnkost) [inline javascript-local-storage/javascript-local-storage-quiz-notes.md:13]: Don't forget the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just need to answer the quiz questions!

File: javascript-local-storage/javascript-local-storage-quiz-notes.md (hunk 1 above)
Comment (kyle-kettler) [reply javascript-local-storage/javascript-local-storage-quiz-notes.md:13]: Sorry, I keep forgetting these! I'll make sure to do better at checking everything before I put in a PR.

**Review by shawnkost — APPROVED** [review]
👍🏻 Regarding the quiz questions, what I like to do every-time I submit a pull request, is go to the "Files changed" tab, and then go through all my files to double check and make sure everything looks good. This way, you'll catch a lot of the things you forgot about (console logs, quiz questions, etc)

**Pull request merged**